field_abilities/frost.js

```diff
@@ -162,7 +162,8 @@ export class FrostFieldPsynergy {
     }
 
     grow_pillar() {
-        this.target_object.interactable_object_sprite.animations.play("frost_pool_pillar", 5, false);
+        let actions = this.data.interactable_objects_db[this.ability_key_name+"_pool"].actions;
+        this.target_object.interactable_object_sprite.animations.play("frost_pool_pillar", actions.frame_rate[1], actions.loop[1]);
```

Comment:
You can remove `actions.frame_rate[1]` and `actions.loop[1]`.

Reply:
Done, thanks!